### ISSUE TITLE
SSH task - use VSTS variables

### ISSUE THREAD

**Issue by sidharthnayyar** [Microsoft/vsts-tasks#3073]
Hi, 

Is it possible to use (inject) VSTS variables into SSH task (commands)?

I have:
SSH-Run:Commands
SSH-Commands:
`echo $PATH;pwd;currenttime=$(date +%Y%m%d_%H%M%S);echo currenttime;buildlabel="$env:RELEASE_ARTIFACTS_Web_BUILDNUMBER";echo buildlabel;backupdir="/usr/Anaconda/envs/web_code/backup/$currenttime.$buildlabel";echo backupdir;sudo cp -R /usr/Anaconda/envs/web_code/web $backupdir;`

But this creates a folder of:
 

> 20161111_121923.:RELEASE_ARTIFACTS_Web_BUILDNUMBER

Maybe the way I'm using the artifacts variable is wrong?

Kind regards
Sidharth

**Comment by AlexRukhlin** [comment]
Hi,

You probably should use RELEASE_ARTIFACTS_WEB_BUILDNUMBER. Also, the following links might help: [TFS Release Management Artifacts Variable not available](http://stackoverflow.com/questions/39404672/tfs-release-management-artifacts-variable-not-available) and [Artifact variables](https://www.visualstudio.com/en-us/docs/release/author-release-definition/understanding-artifacts#variables).

Alex

**Comment by sidharthnayyar** [comment]
Thanks Alex. 

I checked the log and you are right it should be capitalised. Unfortunately both:
`buildlabel="$env:RELEASE_ARTIFACTS_WEB_BUILDNUMBER"`
and 
`buildlabel="$RELEASE_ARTIFACTS_WEB_BUILDNUMBER"`
don't seem to work. The top just returns the text RELEASE_ARTIFACTS_WEB_BUILDNUMBER and the latter gives empty

My understanding was if I'm using inline then I need to replace the "." with "_" and some of the websites also suggest prefixing with "$env:" - which I've tried.

Any other suggestions would be most welcome.

Kind regards
Sidharth
